### DISCUSSION TITLE
Benchmarks against Fastify and Hyper-Express?

### DISCUSSION THREAD

**Discussion by jebarpg** [honojs/hono#2089] [Ideas]
Just curious if there will be any benchmarks against Fastify and Hyper-Express soon?

**Comment by linhub15** [comment]
This might be what you're looking for.
https://github.com/SaltyAom/bun-http-framework-benchmark#results

**Comment by rafaell-lycan** [comment]
@yusukebe @that-ambuj Express v5 is out now. Should be consider having proper benchmarks now?
